logrhythmnextgensiem_connector.py

```diff
@@ -0,0 +1,586 @@
+# File: logrhythmnextgensiem_connector.py
```

Comment:
Could you also add the `pre-commit` config here?
See [MaxMind](https://github.com/splunk-soar-connectors/maxmind) as reference.
After that, please make sure to run:
```
pre-commit install
pre-commit autoupdate
git add .pre-commit-config.yaml
pre-commit run --all-files
```
This is to make sure we have a consistent code styles going forward.

Reply:
Done